src/design-system/InputTextNumeric.tsx

```diff
@@ -49,7 +49,7 @@ const InputTextNumeric: React.FC<Props> = (props) => {
       inputRelativityError={props.inputRelativityError}
       labelPlaceholder={
         props.labelPlaceholder ??
-        (props.type === "number" ? "Enter number" : "Enter a percentage")
+        (props.type === "number" ? "0" : "Enter a percentage")
```

Comment:
Why did we make this particular change?

Reply:
The placeholder text was consistently getting cut off in both the add cases modal and facility detail view now that there are four columns:
![image (2)](https://user-images.githubusercontent.com/29155017/86146118-68ea6b00-babd-11ea-8ad0-7a7178c622a5.png)

This actually could happen in prod with a really small window/screen size, but usually didn't:
![image (3)](https://user-images.githubusercontent.com/29155017/86146206-89b2c080-babd-11ea-916c-c1256cc41659.png)

One alternative that we explored was splitting the placeholder text into multiple lines, but it didn't seem feasible/worthwhile after discussing with @macfarlandian so we did this instead. 

CC: @sychang

Reply:
Got it, I think this makes sense. Thanks!